AGENTS.md

```
@@ -17,7 +17,7 @@ This project builds a reproducible baseball prediction warehouse from free/open
 - Prefer additive migrations and views over destructive schema changes.
 - Keep `README.md` current when commands, setup, or workflow changes.
 - Keep this `AGENTS.md` current when project conventions change.
- - Add and maintain backup procedures for database objects. See `scripts/backup_procedures.sh`.
+ - Add and maintain backup procedures for database objects. See `scripts/utility/backup_procedures.sh`.
```

markdownlint-cli2 (0.22.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)